apps/reaper/lib/reaper/data_slurper/http.ex

```diff
@@ -31,7 +31,7 @@ defmodule Reaper.DataSlurper.Http do
         "Unable to retrieve data for #{ingestion_id}: #{Exception.message(error)}"
       end)
 
-      reraise error, __STACKTRACE__
+      reraise inspect(error), __STACKTRACE__
```

Comment:
this came about because the HTTPError that was boiling up from the Task.Await on line 39 ended up like {{%HTTPError}} for some reason with no real way of removing it without erroring out further when another exception came through. Would gladly change this if solution could be found.